packages/tools/devtools/devtools-core/src/DecomposedContainer.ts

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TypedEventEmitter } from "@fluid-internal/client-utils";
+import type { IAudience } from "@fluidframework/container-definitions";
+import type {
+	AttachState,
+	IContainerEvents,
+	ICriticalContainerError,
+} from "@fluidframework/container-definitions/internal";
+import { ConnectionState } from "@fluidframework/container-loader";
+import type { IEventProvider } from "@fluidframework/core-interfaces";
+import type { IFluidDataStoreRuntime } from "@fluidframework/datastore-definitions/internal";
+
+/**
+ * A lightweight abstraction of {@link @fluidframework/container-definitions/internal#IContainer} that provides
+ * only the essential properties and methods needed for Fluid DevTools functionality.
+ *
+ * @alpha
+ */
+export interface DecomposedIContainer extends IEventProvider<IContainerEvents> {
+	/**
+	 * The audience associated with this container.
```

Comment:
Let's express as much of the property documentation as `{@inheritDoc @fluidframework/container-definitions#IContainer.foo}` as possible to avoid redundancy.